src/hotspot/share/opto/callnode.hpp

```diff
@@ -500,6 +500,7 @@ class SafePointScalarObjectNode: public TypeNode {
                      // states of the scalarized object fields are collected.
                      // It is relative to the last (youngest) jvms->_scloff.
   uint _n_fields;    // Number of non-static fields of the scalarized object.
+  bool _is_auto_box; // is the scalarized object is auto box.
```

Comment:
Typo in comment. Should be something like `// True if the scalarized object is an auto box`

Reply:
Thank you for your review. I'll add this in my next patch.